lib/nexpose/version.rb

```diff
@@ -1,4 +1,4 @@
 module Nexpose
   # The latest version of the Nexpose gem
-  VERSION = '0.9.8'
+  VERSION = '0.9.9'
```

Comment:
need to change this to 1.0.0, but it's on staging/v1 already so i guess it doesn't matter